packages/@aws-cdk/integ-runner/lib/utils.ts

```diff
@@ -75,6 +75,7 @@ export class WorkList<A> {
 
   public done() {
     this.remaining.clear();
+    this.stopTimer();
```

Comment:
This was a bug currently where the runner would not finish immediately if no tests are found.